docs/howto/django/tests.md

```diff
@@ -44,7 +43,7 @@ from procrastinate.contrib.django.models import ProcrastinateJob
 
 def test_my_task(app):
```

Comment:
What is `app` here? Some pytest fixture I am not aware of? Or should it just be `from procrastinate.contrib.django import app`?